apps/src/maze/StepButton.jsx

```diff
@@ -0,0 +1,20 @@
+var msg = require('./locale');
+
+var StepButton = function (props) {
+  var classes = 'launch float-right';
+  if (!props.showStepButton) {
+    classes += ' hide';
+  }
+  return (
+    <button id="stepButton" className={classes}>
```

Comment:
Just kidding, it's fine if this is protected by a parent element.  Just wanted to make sure it was.